xbmc/storage/discs/IDiscDriveHandler.h

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+/*! \brief Represents the state of a disc (optical) drive
+*/
+enum class DriveState
+{
+  DRIVE_OPEN = 0, ///< The drive is open
```

Comment:
`enum class` will always have the enum name which makes the `DRIVE_` prefix redundant.